rfc6962-bis.xml

```diff
@@ -557,6 +562,9 @@ SEQUENCE {
           <spanx style="verb">key_id</spanx> is the SHA-256 hash of the log's public key, calculated over the DER encoding of the key represented as SubjectPublicKeyInfo.
         </t>
         <t>
+          <spanx style="verb">issuer_key_hash</spanx> is the SHA-256 hash of the certificate issuer's public key, calculated over the DER encoding of the key represented as SubjectPublicKeyInfo. This is needed to bind the issuer to the final certificate.
```

Comment:
I just wanted a reminder why we need this.

Although CT does not deal with revocation, clearly there is little point detecting mis-issuance if there's nothing to be done about it, which is the relevance of OCSP.

I guess your suggested text sort of does that. So, with that mod, I'm OK with this update.